app/src/main/java/com/android/unio/ui/association/AssociationProfile.kt

```diff
@@ -422,11 +439,16 @@ private fun AssociationEvents(
   var isSeeMoreClicked by remember { mutableStateOf(false) }
 
   val events by association.events.list.collectAsState()
+  val user by userViewModel.user.collectAsState()
```

Comment:
As user shouldn't be null, maybe it would be better to check if user is null and return if it is. It is what we did for other screens like EventDetails, and it allows us to write `user` instead of `user?` in the rest of the screen.